docs/framework/tools/developer-command-prompt-for-vs.md

```diff
@@ -1,93 +1,135 @@
 ---
-title: "Developer Command Prompt for Visual Studio"
-description: Learn to use the Developer Command Prompt for Visual Studio, which lets you use .NET tools more easily. It automatically sets specific environment variables.
-ms.date: 01/05/2020
+title: Developer command-line tools
+description: Learn how to find and use the Developer Command Prompt for Visual Studio, Developer PowerShell, and Visual Studio terminal, which let you use .NET and C++ tools more easily.
+ms.date: 02/24/2021
+ms.custom: contperf-fy21q3
 helpviewer_keywords:
-  - "command prompt, Windows SDK"
   - "Visual Studio command prompt"
   - "command prompt, Visual Studio"
-  - "SDK command prompt"
-  - "tools [.NET Framework], setting environment variables"
-  - "environment variables, setting for tools"
   - "developer command prompt"
+  - "developer PowerShell"
+  - "Visual Studio terminal"
 ms.assetid: 94fcf524-9045-4993-bfb2-e2d8bad44219
+no-loc: cmdlet
 ---
-# Developer Command Prompt for Visual Studio
+# Command-line shells for developers
 
-Developer Command Prompt for Visual Studio enables you to use .NET Framework tools more easily. It's a command prompt that automatically sets specific environment variables. After opening Developer Command Prompt, you can enter the commands for [.NET Framework tools](index.md) such as `ildasm` or `clrver`.
+Visual Studio 2019 includes two command-line shells for developers:
+
+- **Developer Command Prompt for Visual Studio** - A standard command prompt with certain environment variables set to make using command-line developer tools easier.
+- **Developer PowerShell** - More powerful than a command prompt. For example, you can pass the output of one command (known as a *cmdlet*) to another cmdlet.
```

Comment:
Consider also mentioning that the environment variables in Developer PowerShell are the exact same variables in Developer Command Prompt.